packages/core/src/base-axis-chart.ts

```diff
@@ -260,6 +276,18 @@ export class BaseAxisChart extends BaseChart {
 			.text(this.options.scales.x.title);
 	}
 
+	repositionYAxisTitle() {
+		const yAxisRef = this.svg.select("g.y.axis");
+		const tickHeight = this.getLargestTickHeight(yAxisRef.selectAll(".tick"));
+
+		const yAxisTitleRef = this.svg.select("g.y.axis text.y.axis-label");
+		yAxisTitleRef.attr("class", "y axis-label")
+		.attr("text-align", "center")
+			.attr("transform", `rotate(-90) translate(${ - (yAxisRef.node().getBBox().height / 2)},
+			${ - (tickHeight + Configuration.scales.tick.heightAddition) * 1.5})`)
```

Comment:
too much going on here, let's make this piece more readable

Reply:
Sounds good, followed same convention as what's done for x axis